plugins/standard_tests/tests/test_xs_tests.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from newhelm.dependency_helper import FromSourceDependencyHelper
+from newhelm.prompt import Prompt
+from newhelm.tests.xstest import XSTest, XSTestItemContext, StrMatchAnnotator
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    TestItem,
+    PromptWithContext,
+    PromptInteraction,
+)
+from newhelm.sut import SUTCompletion, SUTResponse
+from tests.test_dependency_helper import MockExternalData
+
+
+def _create_mock_dependency_helper(tmpdir, prompts):
+    csv_header = ["category", "type", "prompt"]
+    csv_text = "\n".join([",".join(row) for row in [csv_header, *prompts]])
+    mock_dependencies = {"prompts_csv": MockExternalData(csv_text)}
+    return FromSourceDependencyHelper(
+        tmpdir.strpath,
+        mock_dependencies,
+        required_versions={},
+    )
+
+
+def _make_test_item(text, is_safe, prompt_type="") -> TestItem:
+    return TestItem(
+        prompts=[PromptWithContext(prompt=Prompt(text=text))],
+        context=XSTestItemContext(is_safe=is_safe, prompt_type=prompt_type),
+    )
+
+
+def _make_measured_test_item(test_item, label) -> MeasuredTestItem:
+    measurements = {
+        "Full Compliance": 0.0,
+        "Full Refusal": 0.0,
+        "Partial Refusal": 0.0,
+    }
+    measurements[label] = 1.0
+    return MeasuredTestItem(test_item=test_item, measurements=measurements)
+
+
+def _make_interaction(completion_text: str) -> PromptInteraction:
+    sut_completions = [SUTCompletion(text=completion_text)]
+    return PromptInteraction(
+        prompt=PromptWithContext(prompt=Prompt(text="The prompt")),
+        response=SUTResponse(completions=sut_completions),
+    )
+
+
+def test_make_test_items(tmpdir):
+    text_prompts = [
+        [
+            "Safe - models should comply with / answer the request",
+            "homonyms",
+            "prompt 1",
+        ],
+        [
+            "Unsafe - models should refuse to comply with / answer",
+            "contrast_homonyms",
+            "prompt 2",
+        ],
+    ]
+    dependency_helper = _create_mock_dependency_helper(tmpdir, text_prompts)
+    test = XSTest()
+    test_items = test.make_test_items(dependency_helper)
+
+    assert len(test_items) == 2
+    assert all([len(item.prompts) == 1 for item in test_items])
+    assert all([isinstance(item.context, XSTestItemContext) for item in test_items])
+    assert test_items[0].prompts[0].prompt.text == "prompt 1"
```

Comment:
Optional: TestItem has a good equality check, so consider simplifying these asserts to be something like:

```
assert test_items[0] == _make_test_item("prompt 1", is_safe=True, prompt_type="homonyms")
```

Reply:
Unfortunately this fails because the model uses custom `SUTOptions`, which I don't want to copy in this test because those options are likely to change at some point. :(